src/main/java/com/lindar/dotmailer/vo/api/NewProgram.java

```diff
@@ -0,0 +1,14 @@
+package com.lindar.dotmailer.vo.api;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class NewProgram {
```

Comment:
This name is a bit confusing as it's not a new program, it's enrolment into a program?

Reply:
Ok, I can use NewProgramEnrolment if you agree

Reply:
Sounds good to me